examples/twittermap/web/public/javascripts/sidebar/controllers.js

```diff
@@ -19,24 +21,45 @@ angular.module("cloudberry.sidebar", ["cloudberry.common"])
       $scope.isHashTagOutdated = false;
     }
 
-    function sendSampleTweetsQuery() {
-      var sampleTweetsRequest = queryUtil.getSampleTweetsRequest(cloudberry.parameters);
+    function sendSampleTweetsQuery(offset=0) {
```

Comment:
This query should contains the filters of field "create_at" and "geo_tag". Like the function `getSampleTweetsRequest(parameters)` in `javascripts/common/query-util` line 239.

Reply:
Also, the filter at "create_at" field can be shoerter, eg. the latest one hours/ 10 minutes, so that this query does not need to take all the tweets in the whole time range, and then sort it.